lib/PostMessageTransport.js

```diff
@@ -58,6 +61,15 @@ define(function (require, exports, module) {
     function send(idOrArray, msgStr){
         var win = _iframeRef.contentWindow;
 
+        // Parse msgStr and check to see if it's a reload command.
+        // If so, trigger launcher.launch and return
+        var msg = JSON.parse(msgStr);
+
+        if(msg.method === "Page.reload" || msg.method === "Page.navigate") {
```

Comment:
The tricky part is that we don't really know what kind of doc we're looking at, because the URL will be a BLOB vs. something we an inspect.  I'm not sure how to handle it yet.  I'm debugging a bit locally.